modules/post/multi/gather/enum_software_versions.rb

```diff
@@ -0,0 +1,139 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Multiplatform Installed Software Version',
+        'Description' => %q{
+          This module, when run against a compromised machine, will gather details on all installed software,
+          including their versions and if available, when they were installed, and will save it into a loot file for later use.
+          Users can then use this loot file to determine what additional vulnerabilites may affect the target machine.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [ 'gwillcox-r7' ],
+        'Platform' => %w[win linux osx bsd solaris android],
+        # 'Platform'      => %w{ android osx win linux bsd solaris },
```

Comment:
It looks like this commented code can be removed.

Reply:
Woops yes that was part of my tests, let me go ahead and remove that so long 👍

Reply:
Resolved with next commit.